lib/matplotlib/widgets.py

```diff
@@ -1086,7 +1098,9 @@ def get_status(self):
         """
         Return a tuple of the status (True/False) of all of the check buttons.
         """
-        return [l1.get_visible() for (l1, l2) in self.lines]
+        return [False if colors.same_color(
+            self._crosses.get_facecolors()[i], colors.to_rgba("none"))
+                else True for i in range(len(self.labels))]
```

Comment:
This is what I tried first too but wouldn't work since `same_color` returns a single boolean and not element-wise comparison.

Reply:
It needs both the arguments to be of the same shape and then just returns `(c1 ==  c2).all()` so can't compare a list of colors with `"none"`

Reply:
Ah oops, that's unfortunate.

This can still be simplified though:
```suggestion
        return [not colors.same_color(color, colors.to_rgba("none"))
                for color in self._crosses.get_facecolors()]
```

Reply:
Fair enough. That's better. Thanks, done!